dev-tools/cloud/terraform/main.tf

```diff
@@ -4,7 +4,7 @@ terraform {
   required_providers {
     ec = {
       source  = "elastic/ec"
-      version = "0.5.1"
+      version = "0.9.0"
```

Comment:
Updated the version following the changes in https://github.com/elastic/fleet-server/pull/3120